index.d.ts

```diff
@@ -1867,19 +1845,26 @@ declare namespace Eris {
     dynamicSplashURL(format?: ImageFormat, size?: number): string;
   }
 
-  export class Invite extends Base {
-    channel: InvitePartialChannel | Exclude<AnyGuildChannel, CategoryChannel>;
+  // If CT (count) is null, it is an invite with metadata, if boolean, it is an invite without metadata
+  export class Invite<CT extends boolean | null, CH extends InviteChannel = InviteChannel> extends Base {
```

Comment:
probably better idea to use a string instead of true/false/null for CT

also CT could have a default to clean up some other code